lib/alerts/util/index.js

```diff
@@ -21,6 +23,17 @@ export const CAUSES = [
   'OTHER_CAUSE'
 ]
 
+export const modes = [
```

Comment:
Could this simply be an array of strings?

Reply:
I'd rather be explicit with the values.

Reply:
I suppose this could future-proof against new modes that would skip a number or two.